### PR TITLE
Fix missing description on PyPI

### PR DESCRIPTION
**Pull request by Kuniwak**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,13 +37,20 @@ def get_version():
...
+
 VERSION = get_version()
 
 
 setup(
     name='vim-vint',
     version=VERSION,
     description='Lint tool for Vim script Language',
+    long_description=read_readme(),
+    long_description_content_type="text/markdown",
```
Comment (blueyed) [inline setup.py]: It's nice that markdown is supported by now, but why do you convert the existing docs?
Do you prefer Markdown?
(just curious)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+$ vint --color --style ~/.vimrc
+```
+
+And you can see all available options by using `--help`:
+
+```console
+$ vint --help
+usage: vint [-h] [-v] [-V] [-e] [-w] [-s] [-m MAX_VIOLATIONS] [-c]
+            [--no-color] [-j] [-t] [--enable-neovim] [-f FORMAT]
+            [--stdin-alt-path STDIN_ALT_PATH]
```
Comment (blueyed) [inline README.md]: This should be updated.. `--stdin-alt-path` is wrong.

Comment (blueyed) [reply README.md]: "This" => the console output.

File: setup.py (hunk 1 above)
Comment (Kuniwak) [reply setup.py]: In my memory, I did not know that what markup languages can be used as `long_description`. But I found an official example that using with `long_description_content_type="text/markdown"`, and I was not sure about what MIME types can be represented as reStructuredText (probably it is [`text/x-rst`](http://docutils.sourceforge.net/FAQ.html#what-s-the-official-mime-type-for-restructuredtext-data) but I could not be sure). So I decided to use Markdown.

File: README.md (hunk 2 above)
Comment (blueyed) [reply README.md]: Fixed help in a063b08.

**Comment by blueyed** [comment]
Closing in favor of #308.

**Pull request closed without merging**